### PR TITLE
Completed html-lists exercise.

### PR DESCRIPTION
**Pull request by david-s-song**
html lists

### CODE REVIEW

File: html-lists/index.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  <body>
+    <h1>HTML Lists</h1>
+
+    <h2>Ordered List Example</h2>
+      <ol>
+        <li>Learn to code.</li>
+        <li>Build projects.</li>
+        <li>Get a job.</li>
+      </ol>
```
Comment (codingcodymiller) [inline html-lists/index.html:15]: These don't need to be indented like this, the `ol` is a sibling of `h2`, so they should be at the same level of indentation.

File: html-lists/index.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+    <h2>Unordered List Example</h2>
+      <ul>
+        <li>HTML</li>
+        <li>CSS</li>
+        <li>JS</li>
+        <li>SQL</li>
+        <li>PHP</li>
+      </ul>
```
Comment (codingcodymiller) [inline html-lists/index.html:24]: Same thing here, this should be even with the above `h2` in indentation

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by david-s-song** [comment]
html lists changed

**Comment by david-s-song** [comment]
html lists changed

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**